src/Flicking.ts

```diff
@@ -1397,7 +1397,7 @@ class Flicking extends Component<FlickingEvents> {
       renderer.batchInsert({ index: 0, elements: parseElement(panels.map(panel => panel.html!)), hasDOMInElements: true });
     }
 
-    if (index) {
+    if (index !== undefined) {
```

Comment:
nit) `index != null` would be better for general I think.
As it can both exclude `null` and `undefined`.